DIGNDB.App.SmitteStop.API/appsettings.Development.json

```diff
@@ -4,9 +4,22 @@
     "logsApiPath": "C:\\SmitteStopNoLogs\\"
   },
   "ExposureConfig": {
-    "AttenuationScores": [1, 2, 3, 4, 5, 6, 7, 8],
-    "DurationAtAttenuationThresholds": [57, 68],
+    "AttenuationScores": [ 1, 2, 3, 4, 5, 6, 7, 8 ],
+    "DurationAtAttenuationThresholds": [ 57, 68 ],
     "MinimumRiskScore": 8
   },
-  "SQLConnectionString": "Server=localhost; Database=DigNDB_Smittestop_NO;Trusted_Connection=True;"
+  "KeyValidationRules": {
+    "PackageNames": {
+      "google": "no.fhi.smittestopp_exposure_notification"
+    }
+  },
+  "JwtAuthorization": {
+    "JwtValidationRules": {
+      "ClientId": "smittestopp",
+      "SupportedAlgorithm": "RS256",
+      "Issuer": "http://localhost:5001"
+    },
+    "JwkUrl": "http://localhost:5001/.well-known/openid-configuration/jwks"
```

Comment:
Should this endpoint `http://localhost:5001/.well-known/openid-configuration/jwks` be running automatically with the use of the new package `AnonymousTokens.Server`?
If not then it'd be nice to mention how to set it up. 
Maybe we should consider adding another simple project which would start up this mock?

Reply:
This is started by https://github.com/folkehelseinstituttet/Fhi.Smittestopp.Verification/

Reply:
Great!
Could you please add a comment in the development json file with this link so people will know where to look?

Reply:
I don't think anything is needed to do here. The README.md is currently empty, but as we finalize the questions on Slack, I intent to write what's needed for a full local development environment,

Reply:
I'll resolve this and create and issue to not forget about it.